depends/packages/native_qtbase.mk

```
@@ -0,0 +1,193 @@
+$(package)_config_opts += -no-gif
+$(package)_config_opts += -no-glib
+$(package)_config_opts += -no-gui
+$(package)_config_opts += -no-harfbuzz
+$(package)_config_opts += -no-icu
+$(package)_config_opts += -no-ico
+$(package)_config_opts += -no-iconv
+$(package)_config_opts += -no-journald
+$(package)_config_opts += -no-kms
+$(package)_config_opts += -no-libjpeg
+$(package)_config_opts += -no-libpng
+$(package)_config_opts += -no-libproxy
+$(package)_config_opts += -no-libudev
+$(package)_config_opts += -no-linuxfb
+$(package)_config_opts += -no-mirclient
+$(package)_config_opts += -no-mtdev
+$(package)_config_opts += -no-openssl
+$(package)_config_opts += -no-opengl
+$(package)_config_opts += -no-openvg
+$(package)_config_opts += -no-qpa
+$(package)_config_opts += -no-reduce-relocations
+$(package)_config_opts += -no-sctp
+$(package)_config_opts += -no-securetransport
+$(package)_config_opts += -no-sql-db2
+$(package)_config_opts += -no-sql-ibase
+$(package)_config_opts += -no-sql-oci
+$(package)_config_opts += -no-sql-tds
+$(package)_config_opts += -no-sql-mysql
+$(package)_config_opts += -no-sql-odbc
+$(package)_config_opts += -no-sql-psql
+$(package)_config_opts += -no-sql-sqlite
+$(package)_config_opts += -no-sql-sqlite2
+$(package)_config_opts += -no-system-proxies
+$(package)_config_opts += -no-trace
+$(package)_config_opts += -no-tslib
+$(package)_config_opts += -no-use-gold-linker
+$(package)_config_opts += -no-widgets
+$(package)_config_opts += -no-xcb
+$(package)_config_opts += -no-xcb-xlib
+$(package)_config_opts += -no-xkbcommon
+$(package)_config_opts += -nomake examples
+$(package)_config_opts += -nomake tests
+$(package)_config_opts += -opensource
+$(package)_config_opts += -no-pkg-config
+$(package)_config_opts += -prefix $(host_prefix)
+$(package)_config_opts += -qt-pcre
+$(package)_config_opts += -qt-zlib
+$(package)_config_opts += -reduce-exports
+$(package)_config_opts += -static
+$(package)_config_opts += -v
+
+# required for C++17 support on macOS
+$(package)_config_opts_darwin = QMAKE_MACOSX_DEPLOYMENT_TARGET=$(OSX_MIN_VERSION)
+
+# for macOS on Apple Silicon (ARM) see https://bugreports.qt.io/browse/QTBUG-85279
+$(package)_config_opts_aarch64_darwin += -device-option QMAKE_APPLE_DEVICE_ARCHS=arm64
+endef
+
+define $(package)_fetch_cmds
+$(call fetch_file,$(package),$($(package)_download_path),$($(package)_download_file),$($(package)_file_name),$($(package)_sha256_hash)) && \
+$(call fetch_file,$(package),$($(package)_download_path),$($(package)_qttools_file_name),$($(package)_qttools_file_name),$($(package)_qttools_sha256_hash))
+endef
+
+define $(package)_extract_cmds
+  mkdir -p $($(package)_extract_dir) && \
+  echo "$($(package)_sha256_hash)  $($(package)_source)" > $($(package)_extract_dir)/.$($(package)_file_name).hash && \
+  echo "$($(package)_qttools_sha256_hash)  $($(package)_source_dir)/$($(package)_qttools_file_name)" >> $($(package)_extract_dir)/.$($(package)_file_name).hash && \
```

style nit, extra spaces:
```suggestion
  echo "$($(package)_sha256_hash) $($(package)_source)" > $($(package)_extract_dir)/.$($(package)_file_name).hash && \
  echo "$($(package)_qttools_sha256_hash) $($(package)_source_dir)/$($(package)_qttools_file_name)" >> $($(package)_extract_dir)/.$($(package)_file_name).hash && \
```

---

Addressed in next push.